src/PRo3D.Viewer/Program.fs

```diff
@@ -272,20 +272,22 @@ let main argv =
         let dumpFile =
             match argsKv |> HashMap.tryFind "dump" with
             | Some file -> file
-            | None when Minerva.Config.ShowMinervaErrors ->
-                Log.warn "need dump file ... dump=\"[dumpfilepath]\" -> using defaultPath '.\MinervaData\dump.csv'"
-                @".\MinervaData\dump.csv"
-            | _ -> 
-                @".\MinervaData\dump.csv"
+            | _ -> "dump.csv"
+            //| None when Minerva.Config.ShowMinervaErrors ->
```

Comment:
dumps beside application? is bad for deployed version when installed?

Reply:
minerva code is dead and needs some other cleanup/handling

Reply:
ok